grails-app/views/public/map3.gsp

```diff
@@ -1,197 +1,204 @@
-<%@ page import="au.org.ala.collectory.CollectionLocation" %>
-<html>
-<head>
-    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
-    <meta name="layout" content="${grailsApplication.config.skin.layout}" />
-
-    <title>
-        <g:message code="public.map3.title" /> | ${grailsApplication.config.projectName}
-    </title>
-
-    <r:require modules="bbq,openlayers,map" />
-</head>
-
-<body>
-<div id="page-collections-map" class="nav-datasets">
-    <div id="content">
-        <div id="header" class="page-header">
-            <h1 class="page-header__title">
-                ${grailsApplication.config.regionName}<g:message code="public.map3.header.title" />
-            </h1>
-
-            <div class="page-header__subtitle">
-                <g:message code="public.map3.header.description" args="[grailsApplication.config.projectNameShort, grailsApplication.config.regionName]" />
-            </div>
-        </div>
-
-        <g:if test="${flash.message}">
-            <div class="message">
-                ${flash.message}
-            </div>
-        </g:if>
-
-        <div class="row">
-            <div class="col-5">
-                <div class="section">
-                    <p>
-                        <g:message code="public.map3.description" />.
-                    </p>
+"<html>
+    <head>
+        <meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
+        <meta name="layout" content="${grailsApplication.config.skin.layout}" />
+
+        <title>
+            <g:message code="public.map3.title" /> | ${grailsApplication.config.projectName}
+        </title>
+
+        <r:require modules="bbq, openlayers, map" />
+
+        <r:script>
+            var altMap = true;
+            var COLLECTIONS_MAP_OPTIONS = {
+                serverUrl:   "${grailsApplication.config.grails.serverURL}",
+                centreLat:   ${grailsApplication.config.collectionsMap.centreMapLat?:"-28.2"},
+                centreLon:   ${grailsApplication.config.collectionsMap.centreMapLon?:"134"},
+                defaultZoom: ${grailsApplication.config.collectionsMap.defaultZoom?:"4"}
+            }
+
+            initMap(COLLECTIONS_MAP_OPTIONS);
+        </r:script>
```

Comment:
It's probably down below because it requires dom to be present. And in any case, scripts by default go after the body so they don't delay rendering.